lib/fifo.js

```diff
@@ -187,6 +177,20 @@ class fifo {
     return qc.wait()
   }
 
+  /**
+  Get a call by callerid
+  */
+  getcallbycallerid( callerid ) {
+    for ( var i = 0; i < this._fifos.length; i++ ) {
+      const othercall = this._calls.get( this._fifos[i][0] )
+      if ( othercall
+          && othercall.call.callerid == othercall.call.callerid ) {
+        othercall.call.__position = i
```

Comment:
minor tweak - I included the vars object in a call for anything outside of the call object to be able to play with - i.e. __postion could in the future, conflict with something in the call object in the future.

line 152 of fifo.js uses a variable: options.call.vars.fifo.position - which could be updated before returning it?

Reply:
this is now replaced by new code